### ISSUE TITLE
BNScalePruner removed all batchnorm2d layers, whatever pruning_ratio.

### ISSUE THREAD

**Issue by DiaJB** [VainF/Torch-Pruning#508]
When I used BNScalePruner to prune YOLOv5 pt file. All batchnorm2d layers were removed, whatever pruning_ratio. I can not finetune the pruned model.

<img width="187" height="345" alt="Image" src="https://github.com/user-attachments/assets/c86ae5d5-c34c-4a48-93eb-c9926579898a" />

**Comment by DiaJB** [comment]
> When I used BNScalePruner to prune YOLOv5 pt file. All batchnorm2d layers were removed, whatever pruning_ratio. I can not finetune the pruned model.
> 
> <img alt="Image" width="187" height="345" src="https://private-user-images.githubusercontent.com/100334898/474799172-c86ae5d5-c34c-4a48-93eb-c9926579898a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ0NTA3MzksIm5iZiI6MTc1NDQ1MDQzOSwicGF0aCI6Ii8xMDAzMzQ4OTgvNDc0Nzk5MTcyLWM4NmFlNWQ1LWMzNGMtNGE0OC05M2ViLWM5OTI2NTc5ODk4YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgwNlQwMzIwMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NTQ4ZmNmMmNhMDJjZGIzNjJmN2ZlOGQ3M2NhNTBjMzgwYjcxNjM0ODg3ODI3MDVjZmE1NDNmN2U1YTM3NzY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.fovh6towGXS2LENdLlM2yxTilno10Drc7QmDr4dlTIA">

It's done. I load the original model with function "attempt_load()" which has the param "fuse" that will fuse Conv layer and BN layer, so batchnorm2d layers were removed with conv layers. To solve the problem, just set "fuse=False" or use "torch.load()" to load pt file.
